src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoEnumTrailingCommaCheck.java

```diff
@@ -123,17 +124,16 @@
 
     @Override
     public int[] getRequiredTokens() {
-        return new int[] {TokenTypes.ENUM_CONSTANT_DEF};
+        return new int[] {TokenTypes.ENUM_DEF};
```

Comment:
it is not breaking compatibility, such token is not adjustable by user so users are not affected